tracer/build/_build/Build.Steps.cs

```diff
@@ -2387,6 +2387,7 @@ static IEnumerable<(string Assembly, string Type)> GetTargetAssembliesFromAttrib
                new (@".*DD_GIT_COMMIT_SHA must be a full-length git SHA.*", RegexOptions.Compiled),
                new (@".*Timeout occurred when flushing spans.*", RegexOptions.Compiled),
                new (@".*ITR: .*", RegexOptions.Compiled),
+               new (@".*CircularChannel: .*", RegexOptions.Compiled),
```

Comment:
👀 What are the errors? Why are they expected? If they happen in "normal" execution then we maybe don't want to write them other than in Debug logs otherwise customers get sad...

Reply:
I think I removed the issue, so reverted this file.